src/ORM/Query.php

```diff
@@ -1178,12 +1178,15 @@ protected function _addDefaultSelectTypes(): void
      *
      * @see \Cake\ORM\Table::find()
      */
-    public function find(string $finder, array $options = [])
+    public function find(string $finder, ...$args)
     {
         /** @var \Cake\ORM\Table $table */
         $table = $this->getRepository();
+        if (!$args) {
```

Comment:
This is sad. Those who have overridden `find()` in their models won't realize they will have to add similar `if` block to their method.

Reply:
What is a chance that someone overriden ORM `Query::find()` and used it instead? `Cake\ORM\Query` is more like internal class, isn't it?

Reply:
Not much chance of overridden `Query::find()`. But I see similar `if` block in `Table::find()` as well a table class of test app which has overridden `Table::find()`.